apps/kitchen-sink/src/ensemble/screens/menu.yaml

```diff
@@ -51,6 +51,9 @@ ViewGroup:
       - label: Help
         icon: HelpOutlineOutlined
         page: help
+      - label: Hidden Page
+        icon: HelpOutlineOutlined
+        hidden: true
```

Comment:
Can we use visible for consistency? And by default make it true

Reply:
replaced `hidden` by `visible` and it will be visible by default